src/constants.js

```diff
@@ -362,6 +351,36 @@ export const styles = {
     backgroundColor: "rgba(255,100,100, 1)"
   },
 
+  resultsPanelContainer: {
+    width: "100%",
+    height: "100%",
+    position: "relative"
+  },
+
+  resultsStatement: {
+    float: "left",
+    width: "80%"
+  },
+
+  resultsAccuracy: { float: "left", fontSize: 18, width: "10%" },
+
+  resultsDetailsButtonContainer: {
+    float: "left",
+    textAlign: "center",
+    width: "10%"
+  },
+
+  resultsDetailsButton: {
+    fontSize: 14,
+    padding: "4px 12px",
+    margin: 0,
+    border: "none",
+    cursor: "pointer",
+    borderRadius: 5,
+    backgroundColor: "#61d2eb",
```

Comment:
#0094CA is the standard "blue" for buttons

Reply:
Oh, fwiw, I tried picking a colour that would look different to our feature words on the left, thinking that this might be confusing:

![Screen Shot 2021-04-29 at 11 22 17 AM](https://user-images.githubusercontent.com/2205926/116491461-c1f66d00-a84e-11eb-8ac3-3ea758fcbc56.png)